### PR TITLE
bedtools: update to 2.27.1 (and remove patch), poco: update maintainer

### PR DESCRIPTION
**Pull request by arjanvandervelde**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/poco/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortSystem          1.0
 name                poco
 version             1.8.0.1
 categories          devel
-maintainers         nomaintainer
+maintainers         gmail.com:vandervelde.ag openmaintainer
```
Comment (mf2k) [inline devel/poco/Portfile]: Please add your GitHub username to the maintainers line.

Comment (arjanvandervelde) [reply devel/poco/Portfile]: hi! how do i add it, just @<username> on the same line?

Comment (mf2k) [reply devel/poco/Portfile]: Yes. But you need to join it to your email like this:
`maintainers         {@arjanvandervelde gmail.com:vandervelde.ag} openmaintainer`

Comment (arjanvandervelde) [reply devel/poco/Portfile]: thanks. done.

**Comment by arjanvandervelde** [comment]
thanks

**Pull request merged**